packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -121,17 +123,24 @@ export class UnfurlService {
             await page.setRequestInterception(true);
             page.on('request', (request: any) => {
                 const requestUrl = request.url();
-                if (blockedUrls.includes(requestUrl)) {
+                const urlHost = url.split('/')[2];
+                // Only allow request to the same host
+                if (!requestUrl.includes(urlHost)) {
```

Comment:
Instead of having a blacklist of URLs, we eonly allow requests from the same URL (aka lightdash) , this will block other urls, even stuff in Markdown and loom 
![image](https://user-images.githubusercontent.com/1983672/221130323-db21e2e1-56ed-4693-b0d7-f56848f91870.png)

Reply:
nice idea 💡 
We can improve how we parse the url 
```
const url = new URL('../cats', 'http://www.example.com/dogs');
console.log(url.hostname); // "www.example.com"
```
[reference](https://developer.mozilla.org/en-US/docs/Web/API/URL)